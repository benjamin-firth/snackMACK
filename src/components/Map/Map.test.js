import React from 'react';
import { shallow } from 'enzyme';
import TruckMap from './Map';

describe('map', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow( <TruckMap lat='40.0' long='40.0' trucks={[{}, {}, {}]}/>)
  })
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})