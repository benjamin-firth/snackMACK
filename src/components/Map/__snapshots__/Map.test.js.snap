// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`map should match the snapshot 1`] = `
<Map
  center={
    Array [
      "40.0",
      "40.0",
    ]
  }
  zoom={10}
>
  <ForwardRef(Leaflet(TileLayer))
    attribution="Â© <a href=\\"http://osm.org/copyright\\">OpenStreetMap</a> contributors"
    url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
  />
</Map>
`;
